README.md

```diff
@@ -421,30 +421,30 @@ Note: the released models were trained with `--n_hidden 2048`, so you need to us
 
 ### Training with augmentation
 
-Augmentation is a useful technique for better generalization of machine learning models. Thus, a pre-processing pipeline with various augmentation techniques on raw pcm and spectrogram has been implemented and can be used while training the model. 
+Augmentation is a useful technique for better generalization of machine learning models. Thus, a pre-processing pipeline with various augmentation techniques on raw pcm and spectrogram has been implemented and can be used while training the model. Following are the available augmentation techniques and can been used at the time of training while using these flags in command line options.
```

Comment:
"and can been used at the time of training while using these flags in command line options." -> "that can be enabled at training time by using the corresponding flags in the command line."